addons/base_import/controllers/main.py

```diff
@@ -10,7 +10,7 @@
 
 class ImportController(http.Controller):
 
-    @http.route('/base_import/set_file', methods=['POST'])
+    @http.route('/base_import/set_file', methods=['POST'], type='http')
```

Comment:
Could you explain why we should add it now? Not sure to figure

Reply:
At the beginning of the refactor, we removed the default value for the `type` argument so we had to make sure all routes had a defined type. Later we found a way to restore the default value (there is a comment in `http._genererate_routing_rules() about it) but we didn't *removed* the changes we made elsewhere.

So indeed, this change is basically useless and because it adds a file to the commit for nothing we could remove it.